QuickDeployment/QuickDeploymentDesktopReadme.md

```diff
@@ -102,26 +117,27 @@ choco apikey add --key="'$YourApiKey'" --source="'https://chocoserver:8443/repos
 - Username: admin
 - Password: [REDACTED]
 
-For using Jenkins, please refer to our documentation here: [https://chocolatey.org/docs/how-to-setup-internal-package-repository](https://chocolatey.org/docs/how-to-setup-internal-package-repository). At most, you will need to login to Jenkins, change the password (`By going to People on the Sidebar > Click on admin > Click Configure on the Sidebar, scroll down to change password section`), and enable the pre-configured jobs to run on the schedule of your choosing. Our documentation can assist with ensuring this is done correctly.
+For using Jenkins, please refer to our documentation here: [https://chocolatey.org/docs/how-to-setup-internal-package-repository](https://chocolatey.org/docs/how-to-setup-internal-package-repository).
+At most, you will need to login to Jenkins, change the password (`By going to People on the Sidebar > Click on admin > Click Configure on the Sidebar, scroll down to change password section`), and enable the pre-configured jobs to run on the schedule of your choosing.
+Our documentation can assist with ensuring this is done correctly.
 
 ### Chocolatey Central Management
 
 - Url: [https://chocoserver](https://chocoserver)
 - Username: ccmadmin
 - Password: 123qwe
```

Comment:
Perhaps this should state "Initial Password" so folks don't think we are exposing a security thing.